lisa/sut_orchestrator/azure/features.py

```diff
@@ -684,7 +684,7 @@ def get_all_disks(self) -> List[str]:
             match = disk_pattern.match(disk_raw.group())
             assert match, "fail to get disk label"
             disk_label = match.group("label")
-            disk_array.append(disk_label)
+            disk_array.append(disk_label.strip())
```

Comment:
Please update the raw_pattern like `/sys/block/(?P<label>sd*\w*)`. It can capture all of them without whitespace, and do it once.